tools/migrate_to_bzlmod.py

```diff
@@ -180,15 +200,25 @@ def print_repo_definition(dep):
             repo_def.append(f"  {key} = {value_str},")
     repo_def.append(")")
 
-    header = "----- Repository information for @%s in the WORKSPACE file -----" % dep["original_attributes"]["name"]
-    eprint(header)
     if "definition_information" in dep:
         eprint(dep["definition_information"])
-    eprint("Repository definition:")
-    for line in repo_def:
-        eprint(line)
-    eprint("-" * len(header))
-
+        append_migration_info(f"""
+<details>
+  <summary>Click here to see where and how the repo was declared in the WORKSPACE file</summary>
+
+#### Location
+```python
+{dep["definition_information"]}
+```
+
+#### Definition
+```python
+{"\n".join(repo_def)}
```

Comment:
I'm getting 
```
SyntaxError: f-string expression part cannot include a backslash
```
you can fix by pre-joining the string:
```
        repo_def_str = "\n".join(repo_def)
```

Reply:
Interesting, I don't get that error when running the test. On which repo did you run the migration tool? I'll add the change anyway